feature/google/src/test/java/BillingWrapperCommonTest.kt

```diff
@@ -911,33 +798,16 @@ class BillingWrapperTest {
         }
     }
 
-    @Test
-    fun `getPurchaseType returns UNKNOWN if sub not found and inapp responses not OK`() {
```

Comment:
any test fully deleted from this file should be in one or both of the flavor-specific files..